plugins/azure_ad_admin/icon_azure_ad_admin/actions/get_user_info/schema.py

```diff
@@ -237,7 +237,7 @@ class GetUserInfoOutput(insightconnect_plugin_runtime.Output):
           "order": 41
         },
         "isResourceAccount": {
-          "type": "string",
+          "type": "boolean",
```

Comment:
since we are changing the types, shouldn't this be a backward incompatible change.

Reply:
I think it's backward compatible change. I changed their only types for output. 
Let me know what you think about it @blaxminarayan-r7

Reply:
Even if you change the outputs, the contract is changed. So, it is backward incompatible change.